database/migrations_sqlite/Version20250119214539.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace intranose\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20250119214539 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Google calendar in db';
+    }
+
+    public function up(Schema $schema): void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->addSql('ALTER TABLE orm_clubs ADD COLUMN google_calendar_id VARCHAR(255) DEFAULT NULL');
```

Comment:
Some duplication with the event's google_calendar_id. Is that intended?

Reply:
Maybe the naming isn't right here. I'm talking about the calendar id that serves as identifier when connecting to the calendar. The event's id is to identify the event itself which is published on the calendar. Some renaming needed here ?

Reply:
No I think it's ok!